apps-rendering/src/components/metadata.tsx

```diff
@@ -153,22 +162,22 @@ const tempraryBackgroundStyle = (format: ArticleFormat): SerializedStyles => {
 				${from.desktop} {
 					background-color: ${neutral[97]};
 				}
-				@media (prefers-color-scheme: dark) {
-					background-color: ${neutral[10]};
-				}
+
+				${darkModeCss`
+					background-color: ${background.articleContentDark(format)};
+				`}
```

Comment:
Just double-checking you noticed the comment at the top of this function? 🙂 I think that was from @marjisound's work on this?